### PR TITLE
create public release

### PR DESCRIPTION
**Pull request by arjun-krishna1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,87 @@
...
+- Employee interviews with system hints
+
+### Output
+
+- Single numeric reimbursement amount (float, rounded to 2 decimal places)
+
+### Historical Data
+
+- `test_cases.json` - 1,000 historical input/output examples
+- `test_cases.csv` - Same data in CSV format for easy analysis
```
Comment (alexccl) [inline README.md:35]: this does not appear to be included yet

File: README.md [hunk 2]
```diff
@@ -1,2 +1,87 @@
...
+## Evaluation
+
+Run `./eval.sh` to test your solution against all 1,000 cases. The script will show:
+
+- **Exact matches**: Cases within ±$0.01 of the expected output
+- **Close matches**: Cases within ±$1.00 of the expected output
+- **Average error**: Mean absolute difference from expected outputs
+- **Score**: Lower is better (combines accuracy and precision)
+
+## ubmission
```
Comment (alexccl) [inline README.md:72]: typo

File: PRD.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+- Output must match the legacy system's output with extremely high fidelity
+- System must handle all 1,000 test cases with minimal or zero deviation
+- Hypothesized rules and edge case handling must be clearly documented
+- Known or suspected bugs in the legacy system must be preserved in the output
+
+## Validation & Success Criteria
+
+Your replica will be tested against 1,000 historical reimbursement cases
+
+It will then run in a shadow deployment on 5,000 live reimbursement requests
```
[the remark this answers is not in the document]
Comment (Melbourneandrew) [reply PRD.md:64]: Maybe we can elaborate on a "shadow deployment". Also, I was under the impression that they would be running the code on the 5k samples and submitting them for grading.

File: PRD.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+Success is defined by:
+
+- How closely your system's outputs match the legacy system
+- Your ability to explain discrepancies or special-case logic
+- Clear documentation of all inferred business rules
+- High performance (sub-millisecond response time)
```
Comment (Melbourneandrew) [inline PRD.md:71]: Performance criteria? I think the only success criteria should be how close the outputs match.

File: README.md (hunk 1 above)
Comment (Melbourneandrew) [reply README.md:35]: Confusing. Just give CSV.

File: run.sh.template [hunk 5]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Black Box Challenge - Your Implementation
+# This script should take three parameters and output the reimbursement amount
+# Usage: ./run.sh <trip_duration_days> <miles_traveled> <total_receipts_amount>
+
+# Example implementations (choose one and modify):
+
+# Option 1: Python implementation
```
Comment (Melbourneandrew) [inline run.sh.template:9]: Instead of "Options" maybe we call them "Examples" to emphasize that they could use any language that they want.

File: PRD.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+## Product Description
+
+You will build a replacement reimbursement engine that:
+
+- Accepts the same input parameters (trip duration, miles, receipt total)
+- Produces the same numeric output as the legacy system
+- Matches the system's behavior across a wide variety of scenarios—including edge cases and likely bugs
+- Runs without any external services (in-memory only)
+- Completes each reimbursement calculation in under 1 millisecond
```
Comment (Melbourneandrew) [inline PRD.md:39]: I am against this requirement. No way for us to verify.

File: test_cases.json
[the remark this answers is not in the document]
Comment (arjun-krishna1) [reply test_cases.json:1]: @alexccl / @Alex-Downey  updated the Validation section of the README to clarify that there is a validation set of 5000 that's not included in test cases.json

```
Your submission will be tested against 5,000 additional cases that are not included in test_cases.json.
```

File: PRD.md (hunk 4 above)
Comment (arjun-krishna1) [reply PRD.md]: Removed the other success criteria @Melbourneandrew

File: PRD.md (hunk 6 above)
Comment (arjun-krishna1) [reply PRD.md]: Removed @Melbourneandrew

**Comment by arjun-krishna1** [comment]
Removed @Melbourneandrew , you are correct there is no shadow deployment
There is the public test set with 1000 that they have been given
And 5000 private test cases that they can't see but we test on

https://github.com/8090-inc/black-box-challenge/pull/1#discussion_r2110386429

**Comment by arjun-krishna1** [comment]
Removed CSV reference @Melbourneandrew , there is only a JSON
So kept that reference and removed the other one:
https://github.com/8090-inc/black-box-challenge/pull/1#discussion_r2110388950

**Comment by Ro-Fo** [comment]
i guess since the start is on Saturday at 10AM PT you should make this private ;)
